gm8emulator-wow64/rust-toolchain.toml

```
@@ -0,0 +1,3 @@
+[toolchain]
+channel = "1.77.2"
```

AFAIK, Rust employs the [SemVer versioning scheme](//semver.org), so the `<PATCH>` component of a version number should not be fixed.

```suggestion
channel = "1.77"
```